sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1419,6 +1419,13 @@ object SQLConf {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefault(100)
 
+  val DISABLED_V2_FILE_READS = buildConf("spark.sql.files.disabledV2Reads")
+    .internal()
```

Comment:
Given the overall DSv2 risk, shall we make this `public` and add migration doc officially, @gatorsmile ?

Reply:
the streaming flag is also internal, if we want to change it, let's change them together in a followup PR.